src/NorthwindWebsite/Infrastructure/Repositories/Implementation/CategoryRepository.cs

```diff
@@ -1,13 +1,40 @@
-﻿using NorthwindWebsite.Infrastructure.Entities;
+﻿using Microsoft.EntityFrameworkCore;
+using NorthwindWebsite.Core.CustomExceptions.InfrastructureExceptions;
+using NorthwindWebsite.Infrastructure.Entities;
 using NorthwindWebsite.Infrastructure.Repositories.Interfaces;
 
-namespace NorthwindWebsite.Infrastructure.Repositories.Implementation
+namespace NorthwindWebsite.Infrastructure.Repositories.Implementation;
+
+public class CategoryRepository : BaseRepository<Category>, ICategoryRepository
 {
-    public class CategoryRepository : BaseRepository<Category>, ICategoryRepository
+    public CategoryRepository(NorthwindContext northwindContext)
+        : base(northwindContext)
+    {
+    }
+
+    public async Task<Category> Get(int id) =>
+        await _context.Categories
+            .AsQueryable()
```

Comment:
I forgot to mention: for searching of collection or one single item you should also pay attention for `AsNoTracking()` method. Please, read some information about it, it may really save your performance in future